source/BatchedGeometry.cpp

```diff
@@ -365,16 +366,22 @@ void BatchedGeometry::build()
 void BatchedGeometry::clear()
 {
    //Remove the batch from the scene
-   if (m_pSceneNode)
-   {
-      m_pSceneNode->removeAllChildren();
-      if (m_pSceneNode->getParent())
-         m_pSceneNode->getParentSceneNode()->removeAndDestroyChild(m_pSceneNode);
-      else
-         m_pSceneMgr->destroySceneNode(m_pSceneNode);
-
-      m_pSceneNode = 0;
-   }
+	if (m_pSceneNode)
+	{
+		try
+		{
+			m_pSceneNode->removeAllChildren();
+		}
+		catch (const std::exception& e)

```

Comment:
this is an unrelated change and should be explained - especially if you silently swallow the exception

Reply:
Remnant of debugging the clear() method.  I have since removed the try/catch statement.